scripts/proposals/proposal_04_service_registry_l2_change_manager_gnosis.js

```diff
@@ -0,0 +1,89 @@
+/*global process*/
+
+const { ethers } = require("ethers");
+
+async function main() {
```

Comment:
A proposal script to change the manager for the ServiceRegistryL2 to swap ServiceManager with ServiceManagerToken on gnosis.